src/agents/models/chatcmpl_stream_handler.py

```diff
@@ -53,6 +53,9 @@ class StreamingState:
     refusal_content_index_and_output: tuple[int, ResponseOutputRefusal] | None = None
     reasoning_content_index_and_output: tuple[int, ResponseReasoningItem] | None = None
     function_calls: dict[int, ResponseFunctionToolCall] = field(default_factory=dict)
+    # New fields for real-time function call streaming
```

Comment:
If I understand correctly, this pull request improves the compatibility of Chat Completions model for function call streaming events, right? If yes, focusing on the changes and considering the document changes (actually it uses responses api and it's already available) in a different pull request would be easier for us to check.